content/howto/web-sso.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Enable Web SSO
+description: ""
+summary: "A guide on how to implement web sso in any web application"
+---
+
+Single sign-on (SSO) is the authentication mechanism, which allows users to sign in to different software systems with a single digital identity. After signing into one application user is signed into another application automatically. 
+
+
+## Web SSO
+
+Web SSO is a method of browser-based session management that utilizes browser storage mechanisms like sessionStorage, localStorage, cookies to maintain the user’s session across your applications.
+
+## How does it work in LoginRadius?
+A centralized domain managed by LoginRadius Auth Page ( IDX ) is utilized to perform the authentication. When requested, this centralized domain shares the session with authorized applications.
+
+So that the users logged in to one application automatically logs into other applications, independent of technology, platform, or domain the user is using.
+
+## Configuration
+
+To use LoginRadius Web SSO, make sure the desired domains are whitelisted under Domain Whitelisting in the LoginRadius dashboard.
+To add the application domain, Login to your LoginRadius Dashboard account, from the left navigation panel, click the Configuration and then navigate to the Domain Whitelisting section. Click the down arrow or anywhere within the section.
+
+<div style="text-align:center">
+  <img src="../assets/domain-whitelisting.png" alt="auth_1" />
+</div>
+
+## Setting Up SSO Token
+This section covers how you can manually set a LoginRadius access_token for SSO.
```

Comment:
can we highlight it as a variable using ` `?

Reply:
For now the navigation would not be available for web sso document on prod so I'll make this change in my next PR.